stack.yaml

```diff
@@ -25,6 +25,7 @@ packages:
   - ./plugins/hls-pragmas-plugin
   - ./plugins/hls-module-name-plugin
   - ./plugins/hls-ormolu-plugin
+  - ./plugins/hls-alternate-number-format-plugin
```

Comment:
You need to update other `stack-*.yaml` files too.

Reply:
Will do that.